recipes/samtools/0.1.12/run_test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# create ex1.fa
+printf ">seq1\nCACTAGTGGCTCATTGTAAATGTGTGGTTTAACTCGTCCATGGCCCAGCATTAGGGAGCTGTGGACCCTGCAGCCTGGCTGTGGGGGCCGCAGTGGCTGAGGGGTGCAGAGCCGAGTCAC\n>seq2\nTTCAAATGAACTTCTGTAATTGAAAAATTCATTTAAGAAATTACAAAATATAGTTGAAAGCTCTAACAATAGACTAAACCAAGCAGAAGAAAGAGGTTCAGAACTTGAAGACAAGTCTCT\n" > ex1.fa
+
+# samtools can fail with exit 255 which is not picked up as an error
+samtools faidx ex1.fa || exit 1
```

Comment:
Looking at the code, it seems like `samtools faidx` will actually exit with status code 0 in this case 😕...
But it behaves correctly if we use additional arguments, hence use this instead:
```bash
samtools faidx ex1.fa seq1
```